src/Microsoft.NetFramework.Analyzers/Core/DoNotUseInsecureDtdProcessing.cs

```diff
@@ -630,9 +630,9 @@ private void AnalyzeObjectCreationForXmlReaderSettings(OperationAnalysisContext
             private void AnalyzeVariableDeclaration(OperationAnalysisContext context)
             {
                 var declare = (IVariableDeclarationOperation)context.Operation;
-                foreach (var variable in declare.Variables)
+                foreach (var declarator in declare.Declarators)
                 {
-                    AnalyzeObjectCreationInternal(context, variable, declare.Initializer?.Value);
+                    AnalyzeObjectCreationInternal(context, declarator.Symbol, declarator.Initializer?.Value ?? declare.Initializer?.Value);
```

Comment:
Declarator has an extension method that will do the right thing here: [https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L147](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/Operations/OperationExtensions.cs#L147)